spec/post_spec.rb

```diff
@@ -0,0 +1,28 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
```

Comment:
- [ ] All the custom methods in the post model should have unit tests, kindly make sure to test the `five_most_recent_comments`,  `update_post_counter`, and all other methods in the model